examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+        elif event == "phx_reply":
+            # Handle command responses
+            ref = message.get("ref")
+            status = payload.get("status")
+            response = payload.get("response", {})
+            
+            if status == "ok":
+                logger.debug(f"✅ Command {ref} successful: {response}")
+            else:
+                logger.error(f"❌ Command {ref} failed: {response}")
+    
+    async def subscribe_to_systems(self, system_ids: List[int]) -> dict:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)